src/storage.rs

```diff
@@ -59,6 +65,20 @@ impl Storer for RedactStorer {
         }
     }
 
+    async fn get_collection(&self, path: &str, skip: i64, page_size: i64) -> Result<DataCollection, Rejection> {
+        match reqwest::get(&format!("{}/data?path={}&skip={}&page_size={}", self.url, path, skip, page_size)).await {
```

Comment:
Should we align storage and client routes? i.e. make a PR on storage to have the route be `/data/<path>` with the rest query params.

Reply:
makes sense to me https://github.com/ajpauwels/redact-store/pull/3